onnxscript/ir/serde.py

```diff
@@ -1213,4 +1283,6 @@ def serialize_dimension_into(
     if isinstance(dim, int):
         dim_proto.dim_value = dim
     elif isinstance(dim, (_core.SymbolicDim, _protocols.SymbolicDimProtocol)):
-        dim_proto.dim_param = str(dim.value)
+        if dim.value is not None:
+            # TODO(justinchuby): None is probably not a valid value for dim_param
```

Comment:
Should we do something when it's None?

Reply:
Looks like we are ok doing nothing. The None was from a test case. We should add a pass to check that no dims are None in rewriter produced values. etc.